docs/src/emulation.md

```diff
@@ -109,38 +109,38 @@ class Program
         {
             Headless: False
         });
-        var iphone12 = playwright.Devices["iPhone 12"];
-        await using var context = await browser.NewContextAsync(iphone12);
+        var iphone13 = playwright.Devices["iPhone 13"];
+        await using var context = await browser.NewContextAsync(iphone13);
     }
 }
 ```
 
+<img width="458" alt="playwright.dev website emulated for iPhone 13" src="https://user-images.githubusercontent.com/13063165/220411073-76fe59f9-9a2d-463d-8e30-c19a7deca133.png" />
+
 ## Viewport
 
 The viewport is included in the device but you can override it for some tests with [`method: Page.setViewportSize`].
 
 ```js tab=js-ts
 import { test, expect } from '@playwright/test';
 
-// Run tests in this file with portrait-like viewport.
 test.use({
-  viewport: { width: 600, height: 900 },
+  viewport: { width: 1600, height: 1200 },
 });
 
-test('my portrait test', async ({ page }) => {
+test('my test', async ({ page }) => {
```

Comment:
why remove everything about portrait? I think it was good to have.

Reply:
we should keep it maybe as a separate heading then? People like to test their website in portrait mode with toInsectViewport etc.